suite-native/device/src/components/DeviceImage.tsx

```diff
@@ -12,13 +12,17 @@ const deviceImageMap: Record<DeviceModelInternal, string> = {
     [DeviceModelInternal.T3W1]: require('../assets/t3w1.png'),
 };
 
-type DeviceImageSize = 'normal' | 'large';
+type DeviceImageSize = 'small' | 'normal' | 'large';
 type DeviceImageDimensions = {
     width: number;
     height: number;
 };
 
-const sizeToDimensionsMap = {
+export const sizeToDimensionsMap = {
```

Comment:
please rename this const since it is exported now. The name is very unspecific in context of different files.
```suggestion
export const deviceImageSizeToDimensionsMap = {
```

Reply:
Done.